openedx/core/lib/xblock_utils/__init__.py

```diff
@@ -303,7 +305,7 @@ def sanitize_html_id(html_id):
     return sanitized_html_id
 
 
-@contract(user=User, block=XBlock, view=basestring, frag=Fragment, context="dict|None")
+@contract(user=User, block=XBlock, view=six.string_types, frag=Fragment, context="dict|None")
```

Comment:
```suggestion
@contract(user=User, block=XBlock, view=six.string_types[0], frag=Fragment, context="dict|None")
```

`six.string_types` is always a single-element tuple. In this case, we can just pull out the first element.

It's kinda gross, but it should work.